### ISSUE TITLE
About training and migration

### ISSUE THREAD

**Issue by yayaYsmile** [amaralibey/gsv-cities#12]
Hello, since I am a novice, I have some questions about training. I made an error when running the result main.py file: Missing Logger folder: LOGS/RESNET50/lightning, do I need to change the default path? How? Can I use the MLP feature aggregation method for image retrieval using other datasets?

**Comment by amaralibey** [comment]
Hello @yayaYsmile,

Can you show me the error you got? The code should be able to create a folder names LOGS in which the model weights get saved. Also, try using lower case letter for the backbone architecture (resnet50). 

For MLP feature aggregation, are you referring to MixVPR? If so, this  is the official repo: https://github.com/amaralibey/MixVPR

Best regards,

**Comment by yayaYsmile** [comment]
I wanted to know how to do the training, and when I ran main.py it didn't work, as shown in the figure. The program didn't seem to create the LOGS folder automatically, so I created it manually. In addition, can the feature aggregation method be used for image retrieval through other datasets. Never mind that the RESNET50 is just a clerical error. Thank you for your reply!
![屏幕截图 2023-03-14 102523](https://user-images.githubusercontent.com/106144030/224877680-7e0c9e55-5d5c-442f-a4c3-787506f269f3.png)

**Comment by amaralibey** [comment]
Hello @yayaYsmile,
I don't see an error message in the screenshot.

How many cpu cores do you have? You might need to reduce the num_workers to 4 and batch_size to 40 or 60 and see if the training progresses.

GSV-Cities dataset is intended to train neural networks for the task of retrieval-based visual place recognition. If you want to use the trained model on other retrieval tasks, you can do it, but performance might not be as good as a model trained for your specific image-retrieval task.

**Comment by yicocc** [comment]
![image](https://user-images.githubusercontent.com/51085010/225547397-500ccf53-e2f5-4918-805d-0b0dfcc59d13.png)
![image](https://user-images.githubusercontent.com/51085010/225547485-e259739d-c2b6-460c-ba83-2d7b5feff3c6.png)
![image](https://user-images.githubusercontent.com/51085010/225547560-b89862c2-c6e4-437c-952b-f2842e6a4ccb.png)
Hello,l want to ask  you for  this question ,I think this result is not correct ,but I don't know how to modify.

**Comment by yayaYsmile** [comment]
> 您好@yayaYsmile，我在屏幕截图中没有看到错误消息。
> 
> 您有多少个 CPU 内核？您可能需要将num_workers减少到 4，batch_size减少到 40 或 60，并查看训练是否进展。
> 
> GSV-Cities数据集旨在训练神经网络执行基于检索的视觉位置识别任务。如果要在其他检索任务上使用经过训练的模型，则可以执行此操作，但性能可能不如为特定图像检索任务训练的模型好。

Thank you for your reply. It seems to be a problem with my equipment.

**Comment by yicocc** [comment]
![image](https://user-images.githubusercontent.com/51085010/225551320-c3e76784-0753-4ba4-ad39-6f80a6ce919e.png)
Hello,this my device.And I reduce the num_workers to 4, batch_size to 40 ,this result didn't change.

**Comment by amaralibey** [comment]
Hello @yicocc,

Have you resolved the issue?
It seems like you are currently running in dev mode instead of a training mode. If you wish to run training, you need to modify `fast_dev_run` to `False`, in the `main.py`, or simply comment the line.
